agent/bench-scripts/tests/pbench-specjbb2005/test-56.post

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+_dir="${_testtmp}/specjbb2005"
+rm -rf ${_dir}/*
+rmdir ${_dir}
```

Comment:
Why do this in two steps, instead of just one?
```
rm -rf ${_dir}
```

Reply:
I am always nervous about `rm -rf`.

Reply:
You do realize that if `${_dir}` evaluates to an empty string, your approach is actually _more dangerous_ than mine, right?  😉